reduce_dimension_binary.json

```diff
@@ -67,7 +67,7 @@
         }
     ],
     "returns": {
-        "description": "A data cube with the newly computed values. It has one dimension less than the original data cube, but the remaining dimensions have their resolution and cardinality preserved.",
+        "description": "A data cube with the newly computed values. It is missing the given dimension, the number of dimensions decreased by one. The dimension properties (name, type, labels, reference system and resolution) for all other dimensions have not changed.",
```

Comment:
> It is missing the given dimension, the number of dimensions decreased by one.

It does not contain the given dimension anymore, the number of dimension decreases by one.

> have not changed

remain unchanged